rdtools/analysis.py

```diff
@@ -34,33 +34,19 @@ class RdAnalysis():
         data for analysis
     pv_input : str
         'power' or 'energy' to specify type of input used for pv parameter
-    pvlib_location : pvlib.location.Location
-        Used for calculating clearsky temperature and irradiance
-    clearsky_poa : pd.Series
-        Right-labeled time Series of clear-sky plane of array irradiance
-    clearsky_cell_temperature : pd.Series
-        Right-labeled time series of cell temperature in clear-sky conditions
-        in Celsius. In practice, back of module temperature works as a good
-        approximation.
-    clearsky_ambient_temperature : pd.Series
-            Right-label time series of ambient temperature in clear sky conditions
-        in Celsius
+
     windspeed : pd.Series
         Right-labeled Pandas Time Series or numeric indicating wind speed in
         m/s for use in calculating cell temperature from ambient default value
         of 0 neglects the wind in this calculation
-    albedo : numeric
-        Albedo to be used in irradiance transposition calculations
+
     power_expected : pd.Series
         Right-labeled time series of expected PV power. (Note: Expected energy
         is not supported.)
     temperature_model : str
         Model parameter pvlib.pvsystem.sapm_celltemp() used in calculating cell
         temperature from ambient
-    pv_azimuth : numeric
-        Azimuth of PV array in degrees from north
-    pv_tilt : numeric
-        Tilt of PV array in degrees from horizontal
+
```

Comment:
Having blank lines here but not elsewhere makes the docstring look a little wonky